samples/ClassroomLabs/Modules/Library/Tools/LabCreator.ps1

```diff
@@ -10,7 +10,7 @@ param(
 )
 
 Import-Module ../Az.LabServices.psm1 -Force
-Install-Module -Name ThreadJob -Force
+Install-Module -Name ThreadJob -Scope CurrentUser -Force
```

Comment:
Switching to installing only per user, so we can run this without needing admin permissions on the powershell window